packages/styles/src/placeholders/_schemes.scss

```diff
@@ -1,10 +1,12 @@
+@use 'sass:map';
 @use '../tokens/temp/scheme';
-@use './../mixins/tokens';
+@use '../mixins/supports';
 
-%color-scheme-light {
-  @include tokens.from(scheme.$post-light);
-}
-
-%color-scheme-dark {
-  @include tokens.from(scheme.$post-dark);
+%scheme-fallback-variables {
+  @include supports.light-dark(false) {
```

Comment:
This seems redundant since the `@include %scheme-fallback-variables;` is only used twice, and both times it's already inside a `@include supports.light-dark(false)` rule.

@oliverschuerch, could you check this change?